test/distributions/privatization/runtime/privatizationProfiler.chpl

```diff
@@ -0,0 +1,16 @@
+use PrivatizationWrappers;
```

Comment:
This test is missing a .good file, could you add it & check that start_test on it passes?

Reply:
Meant to add a 'notest' file for that, sorry.